habitat/sims/habitat_simulator.py

```diff
@@ -451,3 +454,18 @@ def distance_to_closest_obstacle(self, position, max_search_radius=2.0):
         return self._sim.pathfinder.distance_to_closest_obstacle(
             position, max_search_radius
         )
+
+    @property
+    def UP(self):
```

Comment:
Let's add this to Simulator API and name it with lower case and name it like up_vector, forward_vector. Otherwise can be misleading with actions.

Reply:
Where do you want these moved to? Do you mean move them to habitat-sim somewhere? Or am I missing what you're saying?

Reply:
I see what you mean now. shortest_path_follower expects a HabitatSim object rather than an arbitrary Simulator object, so I don't know if this makes sense to add to the Simulator API. It is possible that different simulators will need different algorithms for shortest paths.